src/test/java/emissary/util/shell/ExecutrixTest.java

```diff
@@ -24,37 +24,19 @@
 import java.util.stream.IntStream;
 
 import emissary.test.core.UnitTest;
+import org.apache.commons.lang3.SystemUtils;
 import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
 class ExecutrixTest extends UnitTest {
     private Executrix e;
-    private final boolean isWindows = System.getProperty("os.name").contains("Window");
+    private final boolean isWindows = SystemUtils.OS_NAME.contains("Windows");
 
     @Override
     @BeforeEach
     public void setUp() throws Exception {
         this.e = new Executrix();
```

Comment:
I think you may still have to set `this.e.setTmpDir `but you no longer need to check for `~` in a window's path